gallery/src/components/NxAccordion/NxAccordionPage.tsx

```diff
@@ -75,6 +71,21 @@ const NxAccordionPage = () =>
                 opposed to collapsed.
               </NxTableCell>
             </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>HTML <NxCode>&lt;details&gt;</NxCode> Attributes</NxTableCell>
+              <NxTableCell>
+                <a target="_blank"
+                   rel="noopener"
+                   href="https://developer.mozilla.org/en/docs/Web/HTML/Element/details">
+                  HTML details Attributes
+                </a>
```

Comment:
This might also be a good time to make this improvement all around:
```suggestion
                <NxTextLink external href="https://developer.mozilla.org/en/docs/Web/HTML/Element/details">
                  HTML details Attributes
                </NxTextLink>
```

Reply:
> This might also be a good time to make this improvement all around:

Done here: https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/b43f427716ad3ddcece4e61af8ba47f345401396